_posts/2017-07-25-oes-case-study.md

```diff
@@ -0,0 +1,41 @@
+
+**Maureen Earley:**  Members of our team  worked with the 18F Federal Front Door team that was involved in creating the Standards. The Office of Evaluation Sciences is a champion of the Standards because we’re committed to making government easier for people to interact with. We also wanted to make sure our own website was an ambassador of the Standards and all that they stand for.
+
+**Standards team:** Why did you decide to use the Standards?
+
+**Maureen Earley:** We understood why the Standards were created and why good design across government is important. We also didn’t want to recreate the wheel in designing our website. For example, we realized during the design process that our OES logo didn’t meet color contrast requirements for accessibility. But, because we were referencing the Standards we were able to avoid this issue. We wanted to defer to the Standards and the expertise that already exists.
+
+We also have been working with the team running [Federalist](https://federalist.fr.cloud.gov), the open source static website publishing service, building out templates that other teams across government can use. For example, our [Team page](https://oes.gsa.gov/team/) is now a Federalist template that anyone can use.
+
+When we talk with other offices within government who want to redesign their sites, they get very excited about the Standards, and the Federalist platform, because someone has already done the work for them. It makes their job and life a lot easier.
+
+**Standards team:** How did you integrate your work with the Standards?
+
+**Maureen Earley:** We worked with one of 18F’s developers who was doing the various coding elements for us. She redid the back-end of our site to be compliant with the Standards.
+
+**Standards team:** What were the benefits you gained by using the Standards?
+
+**Maureen Earley:** There were several benefits: a) it’s good design, and b) it’s great that we’re contributing to a cohesive branding approach across all of government. Ideally, we will get to a reality that, when you visit a government website, you know you’re on a government site due to the design consistency.
+
+**Standards team:** Is there anything the Standards  team could do to help you in your  efforts?
+
+**Maureen Earley:** I was unaware of the Public Slack channel, but I want to join because of my personal interest in design.  Creating buy-in for the Standards within the government community is really important.
+
+**Standards team:** Advice for other agencies?
+
+**Maureen Earley:** The U.S. Web Design Standards really make life easier for all of us working in government, which in turn can save taxpayer dollars. Web developers (or even teams without web developers!) have a starting ground to work from, you don’t have to come up with your own branding, and it's a proven toolkit of design features that make government services and information online more accessible to the public. for a better government experience.
```

Comment:
Is "for a better government experience" at the end really its own sentence?  If so, the "for" should be capitalized... but it sounds a bit weird because there's no verb in it...